### PR TITLE
Add Let's Go transfer support

### PR DESCRIPTION
**Pull request by KrisXV**
@Zarel

### CODE REVIEW

File: sim/global-types.ts [hunk 1]
```diff
@@ -73,6 +73,7 @@ type PokemonSet = {
  * - S = event, 3rd char+ is the index in .eventPokemon
  * - D = Dream World, only 5D is valid
  * - V = Virtual Console transfer, only 7V is valid
+ * - G = LGPE transfer, only 7G is valid
```
Comment (Zarel) [inline sim/global-types.ts]: I would use 8V for this. For the same reasons VC transfer uses 7V rather than 1V and 2V.

Comment (Zarel) [reply sim/global-types.ts]: I guess using 8G is fine. We're not _super_ low on letters. But if we are one day, we might want to consolidate them.

Comment (KrisXV) [reply sim/global-types.ts]: Yeah it was a typo

**Comment by KrisXV** [comment]
I didn't mean to edit the SSB file; I just did a file-wide find and replace for transfered into transferred because I assumed my typo was the only one.

**Comment by Zarel** [comment]
Whoops, should probably have waited for unit tests.

**Comment by KrisXV** [comment]
It passed locally on my end with fully up-to-date dependencies, so I think you're good

**Pull request merged**